### PR TITLE
[IMP] hr_fleet : Improve the visual of fleet contracts

### PR DESCRIPTION
**Pull request by jugj-odoo**
task - 3247012
---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/fleet/views/fleet_vehicle_cost_views.xml [hunk 1]
```diff
@@ -20,19 +20,22 @@
...
+                        <group>
+                            <field name="ins_ref"/>
                             <field name="cost_subtype_id"/>
                             <field name="insurer_id"/>
-                            <field name="ins_ref"/>
+                            <field name="service_ids" widget="many2many_tags"/>
                             <field name="company_id" groups="base.group_multi_company"/>
                         </group>
-                        <group string="Vehicle Information">
-                            <field name="vehicle_id"/>
```
Comment (sofiagvaladze) [inline addons/fleet/views/fleet_vehicle_cost_views.xml:32]: what about this one? let's make it invisible as well ^^

File: addons/fleet/views/fleet_vehicle_cost_views.xml [hunk 2]
```diff
@@ -41,17 +44,13 @@
                                 <span class="w-25">
                                     <field name="cost_generated" attrs="{'invisible': [('cost_frequency','=','no')]}" widget="monetary"/>
                                 </span>
-                                <field name="cost_frequency"/>
-                            </div>
```
Comment (sofiagvaladze) [inline addons/fleet/views/fleet_vehicle_cost_views.xml:45]: formatting in stable.. is kinda no no..  that will create forward port conflicts :innocent:

**Review by sofiagvaladze — CHANGES_REQUESTED** [review]
@jugj-odoo thanks for your work :raised_hands: 

Some minor comments ^^

For the commit message header, the model name is fleet and not hr_fleet :innocent:

**Comment by kba-odoo** [comment]
robodoo r+

**Pull request closed without merging**